packages/availity-react-kit/project/app/Request/api/requestApi.test.js

```diff
@@ -0,0 +1,45 @@
+import { spy } from 'sinon';
+import chai from 'chai';
+import sinonChai from 'sinon-chai';
+import moxios from 'moxios';
+import { USER_OPTS, getUser } from './requestApi';
+
+chai.use(sinonChai);
+const should = chai.should();
```

Comment:
Is should preferred?
Given that should works by extending Object.prototype, there are some scenarios where should will not work. Mainly, if you are trying to check the existence (or non-existence) of an object.

Reply:
I think should/expect can be used to achieve the same thing

checking existence
> should.exist(thing)

checking non-existence
> should.not.exist(thing)

Reply:
These are probably ok since you are checking for the existence and it will throw if they do not exist.
> Cannot call method exist of undefined
> Cannot call method exist of null

These are not so great, it will throw when something you do not want to exist doesn't exist.
> Cannot call method not of undefined
> Cannot call method not of null